src/js/common/utils/helpers.js

```diff
@@ -66,6 +66,17 @@ export function getInactivePages(pages, data) {
   return pages.filter(page => !isActivePage(page, data));
 }
 
+export function getCurrentFormStep(chapters, location) {
```

Comment:
It looks like this was originally copied from https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/js/common/components/NavHeader.jsx#L12, so it probably makes sense to import this from there as well.